api/src/main/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminServiceImpl.java

```diff
@@ -323,6 +328,13 @@ public void setAdminLockedState(
     DbWorkspace dbWorkspace = getWorkspaceByNamespaceOrThrow(workspaceNamespace);
     workspaceDao.save(dbWorkspace.setAdminLocked(true));
     adminAuditor.fireLockWorkspaceAction(dbWorkspace.getWorkspaceId(), adminLockingRequest);
+
+    try {
+      mailService.sendWorkspaceAdminLockingEmail(
```

Comment:
our standard emailing practice is best-effort / no retries

I don't see a special reason to deviate from the standard here

Reply:
Not related to this PR but i cannot help wondering if the locking request date really has any significance. The email doesn't mention the exact date as selected by Admin, so the correct assumption is locking is happening at the same time as sending email. 
Since we are giving admin capability to select any date, what if they select a future date, should we lock/send email on future date?

Reply:
We can think of this as a **locking request** time and a **locking start** time.

The locking request time comes first.  That is when the RAB made the decision that this workspace was in violation.  Sometime after that, they tell an admin to lock the workspace.  The admin hits the button, and we record the button-pressing time as the locking start time.

The user first experiences the locking at locking-start time, which is the same time that this email is sent.  There's no need to consult the locking-request time for this email.  (unless we want to include some text like "the RAB made a decision on X date")